stable/influxdb/README.md

```diff
@@ -45,6 +45,13 @@ The command removes all the Kubernetes components associated with the chart and
 
 The default configuration values for this chart are listed in `values.yaml`.
 
+#### General
+
+| Parameter | Description | Default |
+|---|---|---|
+| image.repository | Image repository url | influxdb |
+| image.tag| Image tag | 1.7.6-alpine |
+
```

Comment:
We should expand this table with all values in values.yaml...

Reply:
@aliartiza75 Check out https://github.com/helm/charts/blob/master/stable/influxdb/values.yaml and include all those values please...

However under `config:` there are way too many options, maybe skip those and just mention in the descritption or default column to check values.yaml.

Reply:
👏